components/ai_chat/resources/untrusted_conversation_frame/components/assistant_response/tool_event_content_user_choice.tsx

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+import Icon from '@brave/leo/react/icon'
+import classnames from '$web-common/classnames'
+import ConversationAreaButton from '../../../common/components/conversation_area_button'
+import { createTextContentBlock } from '../../../common/content_block'
+import { useUntrustedConversationContext } from '../../untrusted_conversation_context'
+import type { ToolComponent } from './tool_event'
+import styles from './tool_event.module.scss'
+
+const ToolEventContentUserChoice: ToolComponent = (props) => {
+  const context = useUntrustedConversationContext()
+  const content = props.content
+
+  if (props.toolUseEvent.output) {
+    // Tool already completed, don't allow further response
+    content.toolText = (
+      <ConversationAreaButton
+        onClick={() => {}}
+        isDisabled={true}
+        icon={
+          <Icon
+            className={styles.completedChoiceIcon}
+            name='checkbox-checked'
+          />
+        }
+      >
+        <span data-testid={`tool-choice-text-0`}>{props.toolUseEvent.output[0]?.textContentBlock?.text}</span>
```

Comment:
```suggestion
        <span data-testid="tool-choice-text-0">{props.toolUseEvent.output[0]?.textContentBlock?.text}</span>
```

Reply:
`'jsxSingleQuote': true,` in our prettier config too 😄 (although it didn't complain about my copy paste `{``}` here!)